internal/services/stock_exchange/initialise.go

```diff
@@ -0,0 +1,207 @@
+			Quantity:        rand.Intn(100) + 1,
+		}
+		_, err := BuyOrder(buyOrderBody)
+		if err != nil {
+			fmt.Println(err.Error())
+			return
+		}
+
+		//placing sell order
+		orderID = uuid.New().String()
+		rand.Seed(time.Now().UnixNano())
+		idx = rand.Intn(2)
+		order = orderType[idx]
+		min = stock.LTP - int(float64(stock.LTP)*0.01)
+		max = stock.LTP + int(float64(stock.LTP)*0.01)
+		time.Sleep(1 * time.Second)
+		sellOrderBody := OrderRequest{
+			OrderID:         orderID,
+			StockName:       stock.StockName,
+			OrderPlacedTime: time.Time{},
+			OrderType:       order,
+			LimitPrice:      rand.Intn(max-min+1) + min,
+			Quantity:        rand.Intn(100) + 1,
+		}
+		_, err = SellOrder(sellOrderBody)
+		if err != nil {
+			fmt.Println(err.Error())
+			return
+		}
+	}
+	// sleep and run again
+	time.Sleep(120 * time.Second)
+	RandomizerAlgo()
+}
+
+func GetTickers(limit int) (tickers []string) {
+	urlTickers := "https://api.polygon.io/v3/reference/tickers?active=true&sort=primary_exchange&order=asc&limit=" + strconv.Itoa(limit) + "&apiKey=721mkXq0CBNvCMi5iyJ9E1gBRDiFcT8b"
+	req, _ := http.NewRequest("GET", urlTickers, nil)
+
+	res, _ := http.DefaultClient.Do(req)
+
+	defer func(Body io.ReadCloser) {
+		err := Body.Close()
+		if err != nil {
+			fmt.Println(err.Error())
+			return
+		}
+	}(res.Body)
+	body, _ := ioutil.ReadAll(res.Body)
+
+	rawResponse := bytes.NewReader(body)
+	decoder := json.NewDecoder(rawResponse)
+	parsedResponse := Response{}
+	err := decoder.Decode(&parsedResponse)
+	if err != nil {
+		fmt.Println(err.Error())
+		return nil
+	}
+
+	for _, r := range parsedResponse.Results {
+		//fmt.Println(r.Ticker)
+		tickers = append(tickers, r.Ticker)
+	}
+	return tickers
+}
+
+func InitialiseStock(ticker string) {
+
+	stocksURL := "https://api.polygon.io/v1/open-close/" + ticker + "/2021-08-04?adjusted=true&apiKey=721mkXq0CBNvCMi5iyJ9E1gBRDiFcT8b"
+	req, _ := http.NewRequest("GET", stocksURL, nil)
+
+	res, _ := http.DefaultClient.Do(req)
+	defer func(Body io.ReadCloser) {
+		err := Body.Close()
+		if err != nil {
+			fmt.Println(err.Error())
+			return
+		}
+	}(res.Body)
+	body, _ := ioutil.ReadAll(res.Body)
+	rawResponse := bytes.NewReader(body)
+	decoder := json.NewDecoder(rawResponse)
+	parsedResponse := StockFeed{}
+	err := decoder.Decode(&parsedResponse)
+	if err != nil {
+		fmt.Println(err.Error())
+		return
+	}
+	newStock := model.Stocks{
+		StockTickerSymbol: parsedResponse.Ticker,
+		StockName:         parsedResponse.Ticker,
+		LTP:               int(parsedResponse.LTP * 100.0),
+		OpenPrice:         int(parsedResponse.Open * 100.0),
+		HighPrice:         int(parsedResponse.High * 100.0),
+		LowPrice:          int(parsedResponse.Low * 100.0),
+		PreviousDayClose:  int(parsedResponse.PrevClose * 100.0),
+		PercentageChange:  int(100.0 * (parsedResponse.PrevClose - parsedResponse.LTP) / parsedResponse.PrevClose),
+		//DeletedAt:         time.Time{},
+	}
+	if parsedResponse.Ticker != "" {
```

Comment:
Can we keep this check at https://github.com/RohanDudeja/Tradeasy/pull/44/files?file-filters%5B%5D=.go#diff-07ff93aadb2889f63bdaf8d62896c67f64a397c1345ee3576d053cd6d4ff19e8R137